File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow.cpp

#include "SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow.h"
#include "SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow::getHeaderLabelText( ) 
  {
    return this->HeaderLabelText;
  }
  int SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow.cs
/// </filename>


namespace commands
{
    public class SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow
  {
        private string HeaderLabelText;
     
        public     virtual string getHeaderLabelText()
    {
      return this.HeaderLabelText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

